packages/kbn-interpreter/src/public/browser_registries.js

```diff
@@ -18,7 +18,20 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import $script from 'scriptjs';
+
+function loadPath(path, callback) {
```

Comment:
I took this loader code directly from the main app entry point. `scriptjs` doesn't support CSP nonce attributes, and we weren't using the majority of its features anyway.